core/src/channel/mod.rs

```diff
@@ -249,6 +267,37 @@ impl VoiceChannelData {
                         self.process_control_event(ControlEvent::PitchBendSensitivity(sensitivity))
                     }
                 }
+                0x07 => {
+                    // Volume
+                    let vol: f32 = value as f32 / 128.0;
+                    self.control_event_data
+                    .borrow_mut()
+                    .volume = vol
+                }
+                0x0A => {
+                    // Pan
+                    let pan: f32 = value as f32 / 128.0;
+                    self.control_event_data
+                    .borrow_mut()
+                    .pan = pan
+                }
+                0x40 => {
+                    // Damper / Sustain
+                    let damper = match value {
+                        0..=63 => false,
+                        64..=127 => true,
+                        _ => false,
+                    };
+
+                    for key in self.key_voices.iter() {
+                        key.data.borrow().voices.damper = damper;
+                        key.data.borrow().voices.release_next_voice();
+                    }
+
+                    self.control_event_data
+                        .borrow_mut()
+                        .damper = damper
```

Comment:
I feel like this variable actually never gets used, unless I'm wrong. Maybe it's not necessary to have it? Unless you have some future proofing ideas.

Reply:
I initially put that in when I added the code for the damper control event check, when I didn't know how I would use that value.

It is actually not used at the moment and I don't have anything in mind for the future, so I can remove it and maybe add it back if I ever need it.